roles/kubernetes-apps/ingress_controller/ingress_nginx/defaults/main.yml

```diff
@@ -4,6 +4,7 @@ ingress_nginx_host_network: false
 ingress_nginx_service_type: LoadBalancer
 ingress_nginx_service_nodeport_http: ""
 ingress_nginx_service_nodeport_https: ""
+ingress_nginx_service_annotations: {}
```

Comment:
Could you add this line in `inventory/sample/group_vars/k8s_cluster/addons.yml`?

Reply:
@tico88612 I have added a sample